tests/trainer/test_dataloaders.py

```diff
@@ -249,6 +249,8 @@ def test_mixing_of_dataloader_options(tmpdir):
 
 
 def test_train_inf_dataloader_error(tmpdir):
+    pytest.skip('TODO: fix speed of this test')
```

Comment:
rather decorator...

Reply:
same thing...
multiple ways to skin a cat